### PR TITLE
Add a progress bar

### PR DESCRIPTION
**Pull request by tris460**
Related with the [issue 5](https://github.com/BeelanMX/scrapCFEBot/issues/6), that request a progress bar, I implemented a function which gives you the data percentage that has been obtained.
That function is called every time that the function getDataTable is executed.

Improves:

- Show the percentage in the same line
- Show 0% at the start of the process
- When the progress is 100% add a message 'complete'

### CODE REVIEW

File: cfeScraping/index.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ ScrapPage.prototype.closeBrowser = async function () {
     await this.browser.close();
     return;
   } catch (err) {
-    return console.error('Error: ', err);
+    throw console.error('Error: ', err);
```
Comment (wolftrax5) [inline cfeScraping/index.js:38]: the console can be up for the throw, & can throw only err, or only `throw err`

File: cfeScraping/index.js [hunk 2]
```diff
@@ -98,17 +97,25 @@ const tableToArrays = (selector) => {
...
 };
 
+/**
+ * Get the number of rows in the table
+ * @param {string} selector
+ * @returns int
+ */
+
+const getRows = (selector) => {
+  const rows = document.querySelector(sel - ector).innerText;
```
Comment (wolftrax5) [inline cfeScraping/index.js:107]: There is a Typo in querySelector param `sel - ector`

File: cfeScraping/index.js [hunk 3]
```diff
@@ -125,31 +132,32 @@ ScrapPage.prototype.expectedRows = async function () {
...
   time,
-  rowQ,
   // eslint-disable-next-line prettier/prettier
   callback,
 ) {
   try {
-    const exp = await this.validateExpectedRows(rowQ);
+    let exp = await this.expectedRows(rowSelector);
     let data = await this.getDataTable(tableSelector);
+    typeof exp !== 'string' ? (exp = data.length) : exp;
```
Comment (wolftrax5) [inline cfeScraping/index.js:144]: It's ok, but what if for accidend we get a letter or word a,b,c ...

File: cfeScraping/index.js [hunk 4]
```diff
@@ -125,31 +132,32 @@ ScrapPage.prototype.expectedRows = async function () {
...
     console.log('Expected data: ', exp);
     console.log('Getting data...');
 
     while (exp > obt) {
-      await this.progressBar(data.length, rowQ, callback);
+      await this.progressBar(data.length, exp, callback);
       await this.clickButton(nextPageButton, time);
       const newData = await this.getDataTable(tableSelector);
       data = data.concat(newData);
       obt = obt + newData.length;
```
Comment (wolftrax5) [inline cfeScraping/index.js:155]: `obt = obt + newData.length;` is the same as `obt =+ newData.length;`

**Review by wolftrax5 — CHANGES_REQUESTED** [review]

File: cfeScraping/index.js (hunk 2 above)
Comment (tris460) [reply cfeScraping/index.js:107]: I was testing this, and I forgot correct it

**Review by wolftrax5 — APPROVED** [review]

**Comment by tris460** [comment]
Test complete, you can obtain information of some table editing only the /index.js file.

console
doc

**Pull request merged**